lib/Accordion/AccordionStatusContext.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export const AccordionStatusContext = React.createContext();
+
+export const withAccordionStatus = Component => {
+  const WrappedComponent = props => (
+    <AccordionStatusContext.Consumer>
+      {status => (
+        <Component
+          accordionStatus={status && status.status}

```

Comment:
With babel 7 I assume it is feasible now. Would it be possible to use that?
```suggestion
          accordionStatus={status.?status}
```

Reply:
Thanks Victor! Time to start using these nice, new things...  updated.